spec/src/consensus.rs

```diff
@@ -804,6 +804,13 @@ impl Consensus {
                         let remainder_reward =
                             Capacity::shannons(primary_epoch_reward % epoch.length());
 
+                        let mut next_epoch_length = epoch.length();
+                        let epoch_length_from_param =
+                            cmp::max(self.epoch_duration_target() / MIN_BLOCK_INTERVAL, 1);
+                        if next_epoch_length != epoch_length_from_param {
+                            next_epoch_length = self.epoch_duration_target() / MIN_BLOCK_INTERVAL;
+                        }
```

Comment:
simple int div round up:
```suggestion
let next_epoch_length = (self.epoch_duration_target() + MIN_BLOCK_INTERVAL - 1) / MIN_BLOCK_INTERVAL
```

Reply:
if `self.epoch_duration()` is 0, `next_epoch_length` will be 0, this is wrong.

Reply:
How about this?
```rust
let next_epoch_length = 1 + (self.epoch_duration_target() + MIN_BLOCK_INTERVAL) / MIN_BLOCK_INTERVAL;
```

Reply:
I think we have checked `epoch_duration_target` non-zero somewhere already, it is used as divisor in fn `build_genesis_epoch_ext`.